src/algorithms/configuration/maps/map.py

```diff
@@ -158,6 +156,7 @@ def get_move_along_dir(self, direction: List[float]) -> Point:
         :param direction: The true direction
         :return: The :ref:`ALL_POINTS_MOVE_VECTOR` point
         """
+        assert(any(direction))
```

Comment:
Is this assert really necessary?

Reply:
yes; without it, we run into divide by 0 errors. We're essentially making sure that we have no duplicate Trace objects in the A* generation